apps/docs/src/pages/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { Creator } from "nftoupon";
+import { Arbiter, Creator } from "nftoupon";
```

Comment:
why did we import Arbiter one?

Reply:
Sorry I forgot to remove that extra import